src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -97,6 +100,7 @@ export const TravelAidScreenComponent = ({
               ...serviceJourney,
               estimatedCalls: serviceJourney.estimatedCalls,
             }}
+            isTripCancelled={serviceJourneyDeparture.isTripCancelled}
```

Comment:
I service journey isn't actually cancelled, it is the individual calls that can be cancelled. So the `isTripCancelled` on the `ServiceJourneyDeparture` is actually that the estimated call from the `fromQuayId` is cancelled.

Because of that I think it would be better to use the `cancelled` param on the `serviceJourney.estimatedCall`. The reason for that is mainly that we can then present the `cancelled` value for the focused call, and also that we get more up to date data as the `serviceJourney` is fetched continuously, while the `serviceJourneyDeparture` is more static input parameter to the screen.